### ISSUE TITLE
False positive for @objc funcs with an @escaping closure parameter with Xcode 11.4 (beta)

### ISSUE THREAD

**Issue by rudedogdhc** [realm/SwiftLint#3079]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

A method declared `@objc` with a closure parameter declared `@escaping` triggers a `attributes` false positive. Remove the `@escaping` declaration and the rule is no longer triggered.

### Environment

* SwiftLint version: 0.38.2 
* Installation method used: Homebrew

* Which Xcode version are you using: 11.4 (beta)
* Do you have a sample that shows the issue?

```
echo "@objc\ninternal func foo(identifier: String, completion: (() -> Void))" | swiftlint lint --no-cache --use-stdin --enable-all-rules
Done linting! Found 0 violations, 0 serious in 1 file.

echo "@objc\ninternal func foo(identifier: String, completion: @escaping (() -> Void))" | swiftlint lint --no-cache --use-stdin --enable-all-rules
<nopath>:2:10: warning: Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)
Done linting! Found 1 violation, 0 serious in 1 file.
```

**Comment by iliaskarim** [comment]
That's not the output for me. Both commands output the same output,
```
<nopath>:1:17: warning: Explicit ACL Violation: All declarations should specify Access Control Level keywords explicitly. (explicit_acl)
<nopath>:1:17: warning: Explicit Top Level ACL Violation: Top-level declarations should specify Access Control Level keywords explicitly. (explicit_top_level_acl)
Done linting! Found 2 violations, 0 serious in 1 file.
```
on Swiftlint 0.38.2 for me.

**Comment by rudedogdhc** [comment]
@iliaskarim do you have your Xcode tools set to 11.4 beta?

**Comment by iliaskarim** [comment]
> @iliaskarim do you have your Xcode tools set to 11.4 beta?

No, that could be it!
